### DISCUSSION TITLE
[BUG] Laravel Nutgram onInlineQuery not working

### DISCUSSION THREAD

**Discussion by Mobin-khorushi** [nutgram/nutgram#614] [Q&A]
### Nutgram version

4.11.0

### PHP version

8.3

### Framework

Laravel

### Cache driver

file

### Describe the bug

![image](https://github.com/nutgram/nutgram/assets/63199334/9a9f03a0-638a-456c-8a4f-5b1d77f5b0df)
![image](https://github.com/nutgram/nutgram/assets/63199334/80b8eaa2-2566-4762-acaf-024634424834)

i configured my bot to act like this for testing but nothing happens ! My inline text query is not showing or not receiving any updates I'm using webhooks it stays forever like image below 

![image](https://github.com/nutgram/nutgram/assets/63199334/d66b9306-eb24-478e-9579-8769e93c0158)

### Expected behavior

To Show The inlineresults

### Screenshots

provided

### Additional context

Fixed small Typo 
Changed The HJook Orders 
Changed The Result Type

Tried Everything Nothing Works

Little wired thing I noticed ! when ever I try the query after a while its shows bot connecting ... for 0.1 second
Thanks

**Comment by Mobin-khorushi** [comment]
it seems the problem was about 
`//$bot->onMyChatMember(MyChatMemberUpdateHandler::class);`

handler! when I disabled that onInLineQuery, it worked but why should it be like this ?! please update doc its not mentioned anywhere at all


omg this is so unstable ! its not showing again didn't change anything att all ! what is happening ! 

EDIT: Found out middleware is messing with InLineQuery too ! what is wrong why everything is not allowing onInLineQuery to work ?! am I doing something wrong ?!

**Comment by Lukasss93** [comment]
Did you solve it?

**Reply by Mobin-khorushi** [reply]
sorry for late answer as i told yes when I don't use this
$bot->onMyChatMember(MyChatMemberUpdateHandler::class);

onInLineQuery is working
